commands/show-beacons.go

```diff
@@ -33,28 +32,31 @@ func showBeacons(c *cli.Context) error {
 	if c.String("database") == "" {
 		return cli.NewExitError("Specify a database with -d", -1)
 	}
-	res := database.InitResources("")
+	res := database.InitResources(c.String("config"))
 	res.DB.SelectDB(c.String("database"))
 
 	var data []beaconData.BeaconAnalysisView
-	cutoffScore := .7
-	if c.Bool("all") {
-		cutoffScore = 0
-	}
 
 	ssn := res.DB.Session.Copy()
-	resultsView := beacon.GetBeaconResultsView(res, ssn, cutoffScore)
+	resultsView := beacon.GetBeaconResultsView(res, ssn, 0)
```

Comment:
A similar line exists [here](https://github.com/ocmdev/rita/blob/5321fb65da13a98aef543944c2d0bbd06ecc1b6b/reporting/report-beacons.go#L29).  The `GetBeaconResultsView` function definition has the name cutoffScore for its parameter and it's not hard to look that up so I retract my suggestion about leaving the variable.  I guess what's not obvious is that the value 0 is the lowest possible score and thus equates to getting all results.  So would it make sense to make a comment with what the intention is?  e.g. `// Get all beacon results since 0 is lowest possible score`  Or it might make sense to document the possible values for the `cutoffScore` parameter in the `GetBeaconResultsView` [function](https://github.com/ocmdev/rita/blob/5321fb65da13a98aef543944c2d0bbd06ecc1b6b/analysis/beacon/beacon.go#L65)?